core/corehttp/gateway.go

```diff
@@ -89,9 +90,10 @@ func GatewayOption(writable bool, paths ...string) ServeOption {
 			}, headers[ACEHeadersName]...))
 
 		var gateway http.Handler = newGatewayHandler(GatewayConfig{
-			Headers:      headers,
-			Writable:     writable,
-			PathPrefixes: cfg.Gateway.PathPrefixes,
+			Headers:               headers,
+			Writable:              writable,
+			PathPrefixes:          cfg.Gateway.PathPrefixes,
+			FastDirIndexThreshold: int(cfg.Gateway.FastDirIndexThreshold.WithDefault(100)),
```

Comment:
:information_source: :question:  
 I've set the default to  100 to decrease load on public gateways, but lmk if should be bumped to something higher.

I picked it based on the worst case scenario where all 100 child nodes have 256KiB root blocs, which  produces  ~25MiB of additional data to be fetched by the node before it can produce HTML response.

Reply:
Personal opinion: should be 0. But for lack of any better solution. That fine.